### PR TITLE
[WNMGDS-2147] Last modified git commit hook

### PR DESCRIPTION
**Pull request by forrestbaer**
## Summary

- Adds a git commit hook to our repo that will check to see if any themes have been modified and datestamp them if so.

## How to test

1. `yarn clean && yarn install && yarn build`
2. Delete a token line in one of the `packages/design-system-tokens/src/themes` files.
3. `yarn build:tokens`
4. `git add * && git commit -m 'testing commit hook'`
5. Inspect changes to modified file
6. `git reset HEAD~`

## Checklist

- [x] Prefixed the PR title with the [Jira ticket number](https://jira.cms.gov/projects/WNMGDS/) as `[WNMGDS-####] Title`

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -2,3 +2,15 @@
 . "$(dirname "$0")/_/husky.sh"
 
 npx lint-staged
+
+declare -a THEMES
+DATE=$(date)
+THEMES=($(git diff --cached --name-only | grep -i "theme" || :))
```
Comment (pwolfert) [inline .husky/pre-commit:8]: Do we need to include the styles path somehow? Could this bite us if we ever made a file like `themeUtils.ts`?

Comment (pwolfert) [reply .husky/pre-commit:8]: I tested it out, and it time-stamped my TS file 😆

File: .husky/pre-commit [hunk 2]
```diff
@@ -2,3 +2,15 @@
...
+
+if [ ${#THEMES[@]} -ne 0 ]; then
+  echo "📄 Date-stamping modified token theme files."
+  for i in "${THEMES[@]}"; do
+    echo "Last modified on $DATE" | cat - $i > /tmp/out && mv /tmp/out $i
```
Comment (pwolfert) [inline .husky/pre-commit:13]: It writes the file correctly, but it doesn't make this a comment, so it's not a valid CSS file

Comment (forrestbaer) [reply .husky/pre-commit:13]: image

File: .husky/pre-commit (hunk 1 above)
Comment (forrestbaer) [reply .husky/pre-commit]: 🤦🏻

**Review by pwolfert — APPROVED** [review]
LGTM 🤖

**Pull request merged**